dash/testing/browser.py

```diff
@@ -413,6 +413,7 @@ def get_webdriver(self):
             return getattr(self, "_get_{}".format(self._browser))()
         except WebDriverException:
             logger.exception("<<<Webdriver not initialized correctly>>>")
+            return None
```

Comment:
we can move the return statement to outside of the exception, thereby returning a None in all scenarios where the try fails.

Not sure if that is intended though.

Reply:
Ah, that's better, thanks!

Reply:
Though actually, if the try fails for any reason other than `WebDriverException` the error will be thrown so we'll never reach that return. I'll leave it as is.

Reply:
ah yes thats what I had in mind. If there is a possibility that the try block will fail because of other causes besides `WebDriverException`, then we should probably add a generic `except` block at the end, with the relevant logging and return.